complete/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.springframework</groupId>
+    <artifactId>gs-rest-service</artifactId>
+    <version>0.1.0</version>
```

Comment:
We briefly went over versioning, but this is where it is set for your jar. I didn't mention that the suffix "-SNAPSHOT" is used when you are making a jar that you do not plan to release to production, these jars are just used for testing and development.  So set the version to "0.0.0-SNAPSHOT" for now.